README.md

```diff
@@ -3,56 +3,83 @@
+The purpose of this project is to identify any change in the webpage to maximize or increase the result of the matter. An example can be identifying the clicks lesson for a banner ad.
 
 > **Before you start**
+1. [x] Download the code from Git-Hub https://github.com/sufka861/AB-test.git
+2. [x] Cd to the working directory and start the server using the following command in the terminal - $node app.js
+3. [x] Make sure to install npm packages [npm install] as described in the package.json:
+   8.[x] body-parser
+   9.[x] bson
+   10.[x] cookie-parser
+   11.[x] cors
+   12.[x] dotenv
+   13.[x] eslint
+   14.[x] express
+   15.[x] express-async-errors
+   16.[x] geoip-lite
+   17.[x] moment
+   18.[x] mongodb
+   19.[x] mongoose
+   15.[x] morgan
+   16.[x] newrelic
+   17.[x] node-cron
+   18.[x] request-ip
+   19.[x] sinon
+   20.[x] ua-parser-js
+   21. [x] uuid
+   22. [x] validate-date
+   23. [x] winston
+24. [x] Open a browser page and go to https://ab-test-bvtg.onrender.com/
+25. [x] Make sure you have our .env file containing the sensitive info of our system account.
+26. [x] Open a browser page and go to https://ab-test-bvtg.onrender.com/
+
+> **API reference**
+
+ Full documentation of the API calls can be seen at: https://documenter.getpostman.com/view/24139741/2s8Z711C7C
+9. [x] http calls logger - configured in the middleware directory, writes call logs inside logs folder, filename: http.log
+10.[x] error logs - configured in the middleware directory, writes call logs inside logs folder, filename: error.log. operates from the errorHandler middleware (info will follow)
+11.[x] error handler - consists of 3 parts:
+12.[x] express-async-errors package - wraps the controllers in try/catch statement automatically (no implementation needed), and calls the next(error) function in the catch block with the error thrown. the next function will "fall" into the errorHandler.
+13.[x] errorHandler - a function that is used at the end of the server that catches all the errors thrown in the controllers
+14.[x] Custom Errors - in the errors' directory. Classes that extends Error class, with status and message to pass to the error handier at the end. More can be added if we need them.
 
-1. [x] Make sure you have our .env file containing the sensitive info of our system account.
-2. [x] Make sure to install npm packages as described in the package.json:
-   a. Dotenv
-   b. Morgan
-   c. Node-cron
-   d. Node scheduled
-   f. Uuid
-   g. geoip-lite
-   h.ua-parser-js
-   i. request-ip
-3. [x] Download the code from Git-Hub https://github.com/sufka861/AB-test.git
-4. [x] Cd to the working directory and start the server using the following command in the terminal - $node app.js
-5. [x] Open a browser page and go to https://ab-test-bvtg.onrender.com/
-6. [x] Full documentation of the API calls can be seen at: https://documenter.getpostman.com/view/24139741/2s8Z711C7C
-
-> **Basic Information**
 
+> **Instructions**
```

Comment:
this should be explaining a bit that we are part of a whole system, and are the logic. so the system is based on interacting with different teams UI and SDK